src/documents/activities/FunctionalMashup.html.md.handlebars

```diff
@@ -7,24 +7,32 @@ category: Activities
+<figure>
+[![An image showing a large selection of cards spread out on the floor, each with a different magazine image pasted on.](/images/object_cards.jpg)](/images/object_cards.jpg)
+<figcaption>
+*Figure 1:* Example of a collection of object cards created by pasting random magazine images onto sheets of paper.</figcaption>
+</figure>
+
 
 If desired and available you can provide additional tools and materials that can be used together with the object cards to build prototypes (e.g. scissors, glue, tape, pipe cleaners, etc.).
 ### Steps
-* Break into smaller groups (3 to 5 people per group is usually best) 
-* Each group selects multiple cards from the main deck (the larger group can decide together how many cards/objects each group should select).  
-* Each group decides on a random function or purpose. For example, eating utensils, a communication device, a lock, a fashion item, a vehicle, etc.
-* The challenge for each group is to use their object cards to create something that fulfills the selected function. The group can agree on specific rules if desired.  
+* Break into smaller groups (3 to 5 people per group is usually best)
+* Keeping the objects on the cards hidden, each group selects the same number of cards from the main deck (this number can be decided by the larger group).
+* Each group then chooses or is assigned one function from a collection of functions - these can be determined by the facilitator ahead of time or collectively by the group. The functions can be realistic or can be more experimental and exploratory. For example: communicating with a neighbour during a power failure, sending money to relatives overseas, taking your newborn baby to work, walking your dog in an airplane, looking after your parents remotely, etc.
+* The challenge for each group is to combine and use all of the objects on their cards to create something that fulfills the selected function. One way to approach this might be to tell a story about how your objects come together to fulfill the function.
 
  **Tip**: Brainstorm different approaches and don’t limit your ideas to what is technically possible.
- **Tip**: If there are members in your group that require audio description, be sure to describe the object cards and the process as you move forward. Allow for tactile exploration where appropriate.  
- **Tip**: You can prototype or communicate your solution/concept/idea in whatever way that works best for your group (e.g. sketching, building, describing, etc).   
- **Tip**: In addition to the object cards, you can use the available materials to prototype your idea.     
+
+ **Tip**: If there are members in your group that require audio description, be sure to describe the object cards and the process as you move forward. Allow for tactile exploration where appropriate.
```

Comment:
There are 4 tips in a row. Would it make sense to just have a single **Tip**: and have a list below it?

Reply:
Maybe... but I do like the individual emphasis on each one. We have this in other places too (like Collaborative Art Activity). In some cases the tips are scattered throughout the text and in other cases they are all together like this. I think these particular ones started out scattered throughout the bullet points but it seemed clearer/cleaner to me to put them all at the end of the list.